proto/buf.yaml

```diff
@@ -12,9 +12,27 @@ lint:
     # "legacy" lib protos.
     - teleport/lib/multiplexer/test/ping.proto
     - teleport/lib/web/envelope.proto
-    # TODO: Take care of these.
-    - teleport/lib/teleterm
-    - teleport/lib/prehog
+  ignore_only:
+    # RPC_RESPONSE_STANDARD_NAME and RPC_REQUEST_RESPONSE_UNIQUE make it impossible to follow
+    # Google's standard methods. https://cloud.google.com/apis/design/standard_methods
+    RPC_RESPONSE_STANDARD_NAME:
+      - teleport/lib/teleterm
+    RPC_REQUEST_RESPONSE_UNIQUE:
+      - teleport/lib/teleterm
+    PACKAGE_DIRECTORY_MATCH:
+      # Renaming the proto package name at this point would be a breaking change.
+      - teleport/lib/prehog
+    # TODO(ravicious): Fix protos and re-enable lib/teleterm lint rules below.
+    # TODO(espadolini): Fix protos and re-enable lib/prehog lint rules below.
```

Comment:
@espadolini I tagged you here since you're reviewing this PR but let me know if I should tag someone else.